front/public/locales/en/stdcm.json

```
@@ -34,7 +34,7 @@
     },
     "stopCalculation": "Stop calculation"
   },
-  "spaceSpeedGraphic": "Space-Velocity graph",
+  "speedSpaceChart": "Space-Velocity graph",
```

Do we remove the `-` to be consistent with french trad ?

Also is `Space Velocity graph` the official use by customers for the speed space chart ? (maybe it should be asked in matrix#names)

```suggestion
  "speedSpaceChart": "Space Velocity graph",
```

---

done